%YAML:1.0
---
W: 1440 # 1280
H: 1080 # 1024
PixelFormat: 0x01080009
ExposureTime: 3000
Gain: 0 #10
Gamma: 0.8

K: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   # sentry_right
   data:  [1.7431351577770004e+03, 0., 7.1950000000000000e+02,
         0., 1.7431351577770004e+03, 5.3950000000000000e+02, 
         0., 0., 1.]

D: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   # Sentry_right
   data:  [-4.2475377922787445e-02, -5.8272420015788112e-02, 0., 0., 1.3408844757190523e+00]

C_translation:
   x: 0.0
   y: 0.0
   z: 0.0
